.gitignore

```diff
@@ -36,3 +36,6 @@ yarn-error.log*
 *.tsbuildinfo
 *.tsbuildinfo
 *.tsbuildinfo
```

Comment:
should we get rid of the duplication?

Reply:
Yes, removed in this commit
[40a980e](https://github.com/upleveled/next-js-portfolio-web/pull/107/commits/40a980e4ffb77cc133f6782756117e42bcc712b4)